pkg/eks/eks.go

```diff
@@ -330,38 +330,46 @@ func (c *ClusterProvider) loadClusterKubernetesNetworkConfig(spec *api.ClusterCo
 
 // ListClusters returns a list of the EKS cluster in your account
 func (c *ClusterProvider) ListClusters(chunkSize int, listAllRegions bool) ([]*api.ClusterConfig, error) {
-	if listAllRegions {
-		var clusters []*api.ClusterConfig
-		// reset region and re-create the client, then make a recursive call
-		authorizedRegions, err := c.Provider.EC2().DescribeRegions(&ec2.DescribeRegionsInput{})
-		if err != nil {
-			return nil, err
-		}
+	if !listAllRegions {
```

Comment:
Please update whatever is in eks_test.go to reflect these changes.

Reply:
The unit tests will continue to work with these changes as the same AWS API calls are still made. However, I'm working on adding a test case that will fail on `main`.